core/src/main/java/pl/agh/edu/management/room/RoomManager.java

```diff
@@ -57,13 +57,18 @@ public Optional<Room> findRoomForClientGroup(ClientGroup group) {
 				.min(Comparator.comparing(room -> room.roomState.isDirty()));
 	}
 
+	private int roomTimeMultiplier(Room room) {
+		return (room.getRank().ordinal() + 1) + (room.capacity.ordinal() + 1) / 2;
```

Comment:
Wydaje mi się że taki sam multiplier będzie dla dwóch pokoi z tym samym rankiem ale capacity 1 lub 2. Czy wtedy nie jest taki sam? Bo jak tak to wtedy to dzielenie ze wzoru bym wyrzucił

Reply:
racja, już poprawione